cmd/tiller-proxy/main.go

```diff
@@ -109,7 +111,7 @@ func main() {
 		log.Fatalf("Unable to connect to Tiller: %v", err)
 	}
 
-	proxy = tillerProxy.NewProxy(kubeClient, helmClient)
+	proxy = tillerProxy.NewProxy(kubeClient, helmClient, listLimit)
```

Comment:
I am using `NewProxy` since the properties of `Proxy` are not exposed so it's not possible to initialize it from a different package:

```
	proxy = &tillerProxy.Proxy{
		kubeClient: kubeClient,
	}
^ unknown field 'kubeClient' in struct literal of type proxy.Proxy (but does have proxy.kubeClient)
```

Also I am exposing `Proxy` to avoid having to pass `kubeClient` and `helmClient` for all the methods, is there a benefit if we don't expose it?